Azure/sonic-buildimage
# Add to support thermal plan in as9716 platform

**- What I did**
Add to support thermal plan in as9716 platform
**- How I did it**
Implement  thermal plan in as9716 platform
**- How to verify it**
1. Remove one fan to check other fan duty cycle to 100%  
2. Put Dut to high temp and check fan speed change 

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
